src/HotChocolate/Core/src/Types/Types/OffsetPaging/ICollectionSlice.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections.Generic;
+
+namespace HotChocolate.Types.OffsetPaging
+{
+    public interface ICollectionSlice
```

Comment:
@ademchenko I am not happy with the name CollectionSlice we may should look for a different one.

@PascalSenn @rstaib do we have better name suggestions?

Reply:
Maybe:
ICollectionPartition
ICollectionChunk
ICollectionSegement
ICollectionSector

Maybe event withouth the Collection part
https://github.com/dotnet/runtime/blob/master/src/libraries/System.Linq/src/System/Linq/IPartition.cs

Reply:
I think ICollectionSegment is a good choice since it is consistent with .net naming ArraySegment. At least it should be clear for library users.

Reply:
Or partition sounds also good.

Reply:
@ademchenko we are going with the segment.